index.bs

```diff
@@ -539,9 +539,8 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 
 			* text/uri-list
 			* image/svg+xml
-			* Custom format [=string/starts with=] "web "("web" followed by U+0020 SPACE) prefix
-				and suffix (after stripping out "web "("web" followed by U+0020 SPACE)) is a valid
-				[=/MIME type=].
+			* Custom format [=string/starts with=] `"web "`("web" followed by U+0020 SPACE) prefix
+				and suffix (after stripping out `"web "`) is a valid [=/MIME type=].
```

Comment:
in the MIME sniffing standard I found:

> "text/html;" is not a [valid MIME type string](https://mimesniff.spec.whatwg.org/#valid-mime-type), though [parse a MIME type](https://mimesniff.spec.whatwg.org/#parse-a-mime-type) returns a [MIME type record](https://mimesniff.spec.whatwg.org/#mime-type) for it identical to if the input had been "text/html".

So do we really want to require this to be a valid mime type? The "new ClipboardItem" steps, as well as the algorithm that's touched below, we don't require validity, just that parsing succeeds.